extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css

```diff
@@ -95,6 +99,115 @@
   border-bottom-right-radius: 50% !important;
 }
 
+.i-amphtml-story-interactive-slider-average-indicator {
+  transform: translate(-50%, -50%) scale(0) !important;
```

Comment:
Please add a transition to the transform. In this way the indicator will scale in as it fades in and follow the same pattern as the text indicator.

This may feel like a very small detail but they eye feels these things, especially since the text indicator is animating so close to it.

comparison at half speed (the timing is not completely lined up).
Top is with scale, bottom is without:

https://user-images.githubusercontent.com/3860311/128379050-395c3449-46be-4e18-a640-071dd9982b45.mp4